homeassistant/components/owntracks/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 from .config_flow import CONF_SECRET
 
-REQUIREMENTS = ['libnacl==1.6.1']
+REQUIREMENTS = ['PyNaCl==1.3.0']
```

Comment:
Is this a dropin replacement?

Reply:
Extremely minor API changes, but both libraries are just wrappers around `libsodium`. `libnacl` is older, has a smaller developer base and missing some niceties that are already built into `libsodium` in more recent version such as native base64 support.

I didn't want to be using two different wrappers around `libsodium` which is why I migrated OT to use `PyNaCl`.